js/meetup/Player.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { IoCloseOutline } from 'react-icons/io5';
+import ReactPlayer from 'react-player';
+
+export default function Player(props) {
+  const [modal, setModal] = useState(false);
+
+  const toggleModal = () => {
+    setModal(!modal);
+  };
+
+  return (
+    <span className="player">
+      <a onClick={toggleModal}><i className="icon-youtube-play"></i>  Video</a>
+      {
+        modal ? (
+          <section className="modal__bg">
```

Comment:
Since we use dasharized naming of css classes everywhere else, let's try to be consistent with css naming convention.

Reply:
Addressed following BEM conventions - https://en.bem.info/methodology/naming-convention/